### DISCUSSION TITLE
How to print out LazyEvaluatedKernelTensor

### DISCUSSION THREAD

**Discussion by VhaoYan** [cornellius-gp/linear_operator#28] [Q&A] [answered]
Hi,  I have defined a scaleKernel which returns a covariance tensor as type LazyEvaluatedKernelTensor. I was wondering if there is a way that I can print out this matrix? I looked through the code but cannot find anything  (maybe I miss it). Thank you so much!!

**Comment by Balandat** [comment, marked as answer]
You can use `to_dense()` to make it into a standard tensor.

**Reply by Balandat** [reply]
I guess this is technically a gpytorch discussion not a linear_operator one.

**Reply by VhaoYan** [reply]
Thank you for replying. 

But somehow the "pip install linear_operator" is not working. I checked from PyPI https://pypi.org/project/linear-operator/ which states "linear-operator" not "linear_operator", but still can not be installed from google colab

**Reply by Balandat** [reply]
`linear_operator` requires python 3.8 or newer, and I believe the standard colab env uses python 3.7.

**Reply by Balandat** [reply]
There may be some way to upgrade the python version in a colab session, but I'm not sure about that.
